service/controller/v13/version_bundle.go

```diff
@@ -17,6 +17,41 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Added CA public key into trusted user keys for SSO ssh.",
 				Kind:        versionbundle.KindAdded,
 			},
+			{
+				Component:   "aws-operator",
+				Description: "Fix volume attachments while upgrading.",
```

Comment:
This should be Fixed as the message should be past tense. Would be good to align with `fmt`.

https://github.com/giantswarm/fmt/blob/master/releases/how_write_changelog_message.md

I find that page super helpful because I always forget all this! ;)

Reply:
Done, thanks for the link!